docs/devguide.md

```diff
@@ -266,5 +266,5 @@ the Kubernetes cluster as third party resources.
 
 ## Demo walkthrough
 
-Check out the [walk-through](walkthrough.md) for a detailed guide of an example
+Check out the [introduction](./introduction.md) for a detailed guide of an example
```

Comment:
Similar to the comment from the `walkthrough/README.md`; I would link to the 1.7 walkthrough doc here, and then have a separate little sub-section above here for the intro.

Reply:
As with https://github.com/kubernetes-incubator/service-catalog/pull/1163#discussion_r135657205, I updated the language around this link because I want to funnel people through the intro doc.